UI/src/locales/ar_EG.json

```diff
@@ -2,166 +2,72 @@
     "30": "30",
     "60": "60",
     "90": "90",
-    "# Invoices": "",
-    "% Dep.": "",
-    "(+) Salvage Value": "",
-    "(-) Accum. Dep.": "",
-    "(=) NBV": "",
-    "--FORMAT--": "",
-    "--TEMPLATE--": "",
-    ".": "",
-    "1099-INT": "",
-    "1099-MISC": "",
-    "A return authorization must be obtained from [_1] before goods are returned. Returns must be shipped prepaid and properly insured. [_1] will not be responsible for damages during transit.": "",
-    "AIM": "",
     "AP": "موردين",
     "AP Aging": "كشف حساب موردين",
-    "AP Invoice": "",
-    "AP Invoices": "",
     "AP Outstanding": "ارصدة موردين متبقية",
     "AP Transaction": "حركة حساب مورد",
     "AP Transactions": "حركات حساب الموردين",
-    "AP Voucher": "",
-    "AP account": "حساب مورد",
```

Comment:
Hmm. If I understand correctly, the change is supposed to remove all *untranslated* strings. But this one (and several others) *is* translated.

Reply:
"AP Voucher" is not translated in `ar_EG` so it is removed in JSON to default to the english version.
"AP account" is not used anymore in our code, so it is removed complelety